geopandas/base.py

```diff
@@ -1140,13 +1140,13 @@ def geom_almost_equals(self, other, decimal=6, align=True):
         dtype: geometry
 
 
-        >>> s.geom_almost_equals(Point(0, 1), decimal=2)
+        >>> s.geom_equals_exact(Point(0, 1), decimal=2)
         0    False
         1    False
         2     True
         dtype: bool
 
-        >>> s.geom_almost_equals(Point(0, 1), decimal=1)
+        >>> s.geom_equals_exact(Point(0, 1), decimal=1)
         0    False
```

Comment:
These need to be reverted, they are the documentation for `geom_almost_equals`, which still needs to be accurate until `geom_almost_equals` is completely removed